tests/test_other.py

```diff
@@ -7985,10 +7985,11 @@ def args_to_filename(args):
     sent = [x for x in sent if x]
     sent.sort()
 
-    sent_file = expected_basename + '.sent'
-    sent_data = '\n'.join(sent) + '\n'
-    self.assertFileContents(sent_file, sent_data)
-    self.assertEqual(len(sent), expected_sent)
+    if expected_imports is not None:
+      sent_file = expected_basename + '.sent'
+      sent_data = '\n'.join(sent) + '\n'
+      self.assertFileContents(sent_file, sent_data)
+      self.assertEqual(len(sent), expected_sent)
```

Comment:
I don't know much about this code, but does this code depend on `expected_imports`? There's no use of the variable here

Reply:
Yeah, it seems unrelated, but basically "sent" is the stuff on the JS side being sent to wasm. If we don't care about wasm imports, we don't care about what is sent to them either.